server/app/Http/Controllers/UsersController.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\UsersRequest\UserCreateRequest;
+use App\Http\Requests\UsersRequest\UserUpdateRequest;
+use App\User;
+use Hash;
+
+class UsersController extends Controller
+{
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+
+        $campos = ['id', 'name', 'email', 'created_at'];
+
+        $porPag = $request->input("porPag");
+
+        $Users = new User();
+        $lista = $Users::select($campos)->paginate($porPag);
+
+        $respuesta = $lista;
+
+        return $respuesta;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(UserCreateRequest $request)
+    {
+        $name = $request->input('name');
+        $email = $request->input('email');
+        $pass = $request->input('password');
+        $hash = Hash::make($pass);
+
+        try {
+
+            $User = new User();
+            $User->name = $name;
+            $User->email = $email;
+            $User->password = $hash;
+            $User->save();
+
+            $transaccion = true;
+        } catch (\Exception $e) {
+            $transaccion = false;
+        }
+
+        if ($transaccion) {
+            $respuesta = ['mensaje' => "El Usuario {$name} se creado con exito"];
```

Comment:
Son solo errores de ortografía, deberíamos tratar de que no lleguen a producción ya que es texto que se muestra  al usuario.
```suggestion
            $respuesta = ['mensaje' => "El usuario {$name} se ha creado con éxito"];
```

Reply:
Solucionado en 78c15191cedf9cb693845c39b7103b481a27a563 ✔